lib/src/base-styles/_nx-pagination.scss

```diff
@@ -38,35 +49,35 @@
   }
 }
 
-.nx-btn--pagination {
+.nx-btn.nx-btn--pagination {
   @include nx-text-helpers.regular;
-  background-color: transparent;
   border-color: transparent;
   color: var(--nx-color-text);
 
   &:hover {
-    background-color: transparent;
-    border-color: var(--nx-color-interactive-border-hover);
+    background-color: var(--nx-color-interactive-background-hover);
+    color: var(--nx-color-text-dark);
   }
 
   &:focus {
-    border-color: var(--nx-color-interactive-border-focus);
+    @extend %btn-focus-outline;
+    outline-offset: -2px;
   }
 
   &:active {
-    background-color: var(--nx-swatch-indigo-97);
-    border-color: var(--nx-color-interactive-border-hover);
-    box-shadow: none;
+    background-color: var(--nx-swatch-grey-97);
     color: var(--nx-color-text);
+    outline: none;
   }
 
   &.selected {
-    background-color: var(--nx-swatch-indigo-15);
+    background-color: var(--nx-swatch-blue-40);
     color: var(--nx-color-component-background);
     cursor: default;
```

Comment:
Hmm, when there's only one button, or the first button in pagination when selected and hovered the background is gray and the text is white:

<img width="140" alt="Screenshot 2023-03-10 at 11 46 35 AM" src="https://user-images.githubusercontent.com/10160030/224374617-b543ffc8-3d51-49b9-92c6-cc1c2f693022.png">

<img width="245" alt="Screenshot 2023-03-10 at 11 48 49 AM" src="https://user-images.githubusercontent.com/10160030/224374642-6fa7cd27-c991-4823-a966-b824683ff4a6.png">

Reply:
>Hmm, when there's only one button, or the first button in pagination when selected and hovered the background is gray and the text is white:

Thanks for catching that, I completely missed it. Fixed here: [4d57616](https://github.com/sonatype/sonatype-react-shared-components/pull/1004/commits/4d57616ff2edb59bd55e2857b49975ad3817bfb5)